spec/stacktrace_spec.rb

```diff
@@ -87,4 +87,43 @@
       })
     }
   end
+
+  describe "#vendor_cache" do
```

Comment:
I don't understand this doc string, as there isn't a `vendor_cache` instance method, and the tests don't appear to be related to caching.